flink-python/src/main/java/org/apache/flink/streaming/api/operators/python/AbstractPythonFunctionOperator.java

```diff
@@ -273,12 +271,12 @@ public boolean isBundleFinished() {
 
     /** Reset the {@link PythonConfig} if needed. */
     public void setPythonConfig(PythonConfig pythonConfig) {
-        this.config = pythonConfig;
```

Comment:
Since pythonConfig is an transient variable, we should refactor this method to setConfiguration(Configuration conf)